CONTRIBUTING.md

```diff
@@ -2,6 +2,6 @@
 
 I'm really glad you're reading this, because we need your help to make this project come to fruition.
 
-If you haven't already, come find us on Slack ([#forum-acm](https://coreos.slack.com/archives/CTDEY6EEA)). We want to help answer your questions about Open Cluster Management.
+If you haven't already, come find us on Slack ([#forum-acm](https://coreos.slack.com/archives/CTDEY6EEA)). We want to help answer your questions about the `Open Cluster Management` organization.
```

Comment:
@bcpswope Can you add a note here in the #forum-acm link that indicates this is an "internal RedHat" slack channel only.